aws_lambda_powertools/utilities/idempotency/idempotency.py

```diff
@@ -98,6 +105,7 @@ def __init__(
         persistence_store : BasePersistenceLayer
             Instance of persistence layer to store idempotency records
         """
+        persistence_store.configure(config)
```

Comment:
@heitorlessa there would have to be a better way than this, just try to see if the UX works.